content/resources/index.md

```diff
@@ -25,6 +25,13 @@ Watch the video on OpenADx here on YouTube: https://youtu.be/7gHLkSzWNKA
 - [Vorteile von Open Source fuer ADAS und autonomes Fahren (only in German available)](https://www.elektroniknet.de/elektronik-automotive/assistenzsysteme/vorteile-von-open-source-fuer-adas-und-autonomes-fahren-154085.html)
 - [Open Source Software Drives Indy Autonomous Challenge Racecars](https://newsroom.eclipse.org/eclipse-newsletter/2021/june/open-source-software-drives-indy-autonomous-challenge-racecars)
 - [Q&A: It’s Time to Take a Closer Look at Eclipse iceoryx](https://newsroom.eclipse.org/eclipse-newsletter/2021/june/qa-it%E2%80%99s-time-take-closer-look-eclipse-iceoryx)
+- [Collaborating with Ecosystem Partners to Accelerate “Autonomous Driving for All” by ADLON](https://www.adlinktech.com/en/Connected-Autonomous-Vehicle-Solutions)
```

Comment:
If possible, please change "ADLON" to "ADLINK"
We can do the change also next time.

Reply:
I made the update before merging!